src/ucp/api/ucp.h

```diff
@@ -2809,7 +2856,7 @@ ucs_status_ptr_t ucp_am_send_nbx(ucp_ep_h ep, unsigned id,
 
 /**
  * @ingroup UCP_COMM
- * @brief Receive Active Message sent with rendezvous protocol.
+ * @brief Receive Active Message by provided data descriptor.
```

Comment:
@tonycurtis `by provided data descriptor` sounds odd to me. Maybe `as defined by data descriptor`